openedx/core/djangoapps/user_api/tests/test_helpers.py

```diff
@@ -108,7 +110,7 @@ def test_to_json(self):
                     },
                     "errorMessages": {
                         "required": "You must provide a value!"
-                    }
+                    },
```

Comment:
Nit: just to keep the git history clean, I prefer to not make small syntactic changes like this that don't impact functionality. Feel free to ignore.